### ISSUE TITLE
Pawns can't haul to Armor Racks if item is in another stockpile with Tick prepatch option enabled

### ISSUE THREAD

**Issue by khamseen** [bbradson/Performance-Fish#60]
After adding PF I ran into an issue where none of my pawns would haul items to my armor racks, took a while but I tracked down which prepatch option is causing it.

Prepatch option Haulables > Tick (Handles things queued for removal by the StoreUtilityPrepatches:TryFindBestBetterStoreCellFor patch)

The issue only appears to happen when the item is in a stockpile that it's allowed to be in, if I uncheck that item from the stockpile it's stored in or delete the stockpile so the item is on the floor then the item will be moved to the armor rack correctly.
Before you ask, it's not a priority issue. I had armor rack on Critical and tried all other priorities on second stockpile holding the items and none of them triggered a pawn to move the item. Also reloading after disabling the Tick prepatch option pawns would instantly move the item from the stockpile without changing any settings. 

Tested with Core, all DLCs, and Armor Racks (by khamenman), plus PF and it's dependencies.

**Comment by KuarKuar** [comment]
Thank you for saving my save file! I ran into the same issue as you, though mine might have been even worse. After removing my cheese from the press, I found I couldn't place it on a storage shelf. Then, my pawns stopped responding, and the game screen started flickering like crazy. I honestly thought my save file was completely corrupted. 

Luckily, I came across your solution and decided to give it a shot. I disabled the Tick Prepatch option as you suggested (and before that, I had also turned off options like TryFindBestBetterStoreCellForWorker, TryFindBestBetterStoreCellFor, AllowedToAccept, and Improve Hauling Accuracy). It worked perfectly—you saved my game! You’re a lifesaver!
